pipelines/base/peribolos-dump-config.yaml

```diff
@@ -14,7 +14,29 @@ spec:
   - name: GIT_NAME
     type: string
   steps:
-    - name: dump-and-push-config
+    - name: dump-config
+      image: quay.io/skopecky/peribolos-fix
+      volumeMounts:
+        - mountPath: /var/lib/shared
+          name: shared-data
+      env:
+        - name: GITHUB_TOKEN
+          valueFrom:
+            secretKeyRef:
+              name: peribolos-$(params.INSTALLATION_ID)
+              key: token
```

Comment:
I think this is unnecessarily complicated.

1. You're using a key from a secret named `token`.
2. You use its value and assign it to environment variable
3. Later in the script you use that environment variable and dump it into a file


Why not mount it from the secret as a file directly?

https://kubernetes.io/docs/concepts/configuration/secret/#using-secrets-as-files-from-a-pod